src/config_design.config

```diff
@@ -0,0 +1,126 @@
+%% Cloudant_benchmark config changes design review 
+
+%% ----------------------------------------------------------------------------
+%% CONFIGURATION TEMPLATES: (Fogbugz 62351)
+%% ----------------------------------------------------------------------------
+
+{config_templates, ["basho_bench", "cloudant_http", "http_driver"]}.
+
```

Comment:
I'll file a separate ticket for the idea and put it into the backlog but as we get a more complex richer environment I think it will be worthwhile. We can outgrow the casual "sample config" idea pretty quickly, especially when we start to have overlapping common segments and using multiple "applications" in the same run. 

https://cloudant.fogbugz.com/f/cases/63072/Use-configuration-templates-to-demonstrate-document-validate-config-params-for-multiple-applications